### PR TITLE
feat: add carbon telemetry

### LINKED ISSUE #305: Onboard to `@carbon/telemetry`
https://www.npmjs.com/package/@carbon/telemetry

### PR DESCRIPTION
**Pull request by lee-chase**
Closes #305

Add Carbon telemetry to cloud-cognitive and cloud-cognitive-experimental packages.

#### What did you change?

package.json in cloud-cognitive and cloud-cognitive-experimental packages.

#### How did you test and verify your work?

Ran ci-check

### CODE REVIEW

**Review by davidmenendez — APPROVED** [review]

File: packages/cloud-cognitive/package.json [hunk 1]
```diff
@@ -37,13 +37,15 @@
...
     "test": "bundler check 'src/**/*.scss'"
   },
   "peerDependencies": {
     "react": "^16.13.1"
   },
   "dependencies": {
     "@carbon/ibm-cloud-cognitive-experimental": "^0.26.8",
-    "@carbon/ibm-cloud-cognitive-security": "^0.4.11"
+    "@carbon/ibm-cloud-cognitive-security": "^0.4.11",
+    "@carbon/telemetry": "^0.0.0-alpha.6"
```
Comment (dcwarwick) [inline packages/cloud-cognitive/package.json:49]: This adds a new dependency. Have we done the review checks?

**Review by dcwarwick — CHANGES_REQUESTED** [review]
## Review of dependencies

@carbon/telemetry is being added as a new dependency:

- [ ] Verify the dependency has previously been pedigree reviewed
- [ ] Verify the dependency doesn't contain any vulnerabilities
- [ ] Verify the dependency bundle size is acceptable
- [ ] Verify the dependency is actively maintained
- [ ] Verify license conditions are met e.g. forwarding of copyright if necessary.

File: packages/cloud-cognitive/package.json (hunk 1 above)
Comment (lee-chase) [reply packages/cloud-cognitive/package.json:49]: Good point.
1. It's Carbon not sure this is applicable.
2. What is the process for this? https://www.npmjs.com/package/@carbon/telemetry?activeTab=dependencies
3. 27.3 kB unpacked, but not added to our component dist.
4. Spoke with Josh earlier and it is still alive.
5. License is the standard Carbon Apache-2.0 License.

Comment (dcwarwick) [reply packages/cloud-cognitive/package.json:49]: Agree on (1).

For (2), is `yarn audit` sufficient? If so, I can confirm that `yarn audit` does not report any vulnerabilities for @carbon/telemetry.

Comment (lee-chase) [reply packages/cloud-cognitive/package.json:49]: Happy to go with 'yarn audit' as a step. We should probably have an explainer for these checkboxes in a doc somwhere.

**Review by dcwarwick — APPROVED** [review]

**Pull request merged**